examples/lighting-app/linux/ui.cpp

```diff
@@ -290,15 +290,25 @@ void DeviceState::ChipLoopUpdateCallback(intptr_t self)
 
 void DeviceState::UpdateState()
 {
+    if (!chip::DeviceLayer::PlatformMgr().IsServerRunning())
+    {
+        gUiRunning = false; // main loop stopped, stop UI as well
```

Comment:
Shouldn't whatever code calls StopEventLoopTask() also stop the UI event loop?

Reply:
Updated the code to do so.